ev3dev2/motor.py

```diff
@@ -2561,41 +2563,134 @@ def on_arc_left(self, speed, radius_mm, distance_mm, brake=True, block=True):
         """
         self._on_arc(speed, radius_mm, distance_mm, brake, block, False)
 
-    def _turn(self, speed, degrees, brake=True, block=True):
+    def turn_degrees(self, speed, degrees, brake=True, block=True, error_margin=2, use_gyro=False):
```

Comment:
`_turn` was there in 2.0.0 but this is private so I don't think this would count as a breaking change since it is not part of the public API